project/src/components/favorites-screen/favorites-screen.tsx

```diff
@@ -0,0 +1,170 @@
+function FavoritesScreen(): JSX.Element {
+  return (
+    <div className="page">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <a className="header__logo-link" href="main.html">
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
```

Comment:
```suggestion
                <img 
                  className="header__logo" 
                  src="img/logo.svg" 
                  alt="6 cities logo" 
                  width="81" 
                  height="41" 
                />
```

И вот длинные такие штуки, принято делать вот так, красиво и понятно